src/main/cpp/blaze_util.cc

```diff
@@ -74,6 +76,26 @@ bool GetNullaryOption(const char *arg, const char *key) {
   return true;
 }
 
+std::vector<std::string> SearchNaryOption(const vector<string>& args,
```

Comment:
I _think_ this'd be more clear if we named it `SearchUnaryOptions` or `GetAllUnaryOptionValues`. "Nary" implies the flag has multiple arguments, but what we really want to convey is that we want to find all values of the flag.

Reply:
Sounds good.

Reply:
Renamed.